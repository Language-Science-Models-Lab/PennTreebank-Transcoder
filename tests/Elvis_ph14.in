
(S (NP @(NNP @Elvis) @(NNP @Presley) ) @(VP @(VBZ @is) (ADJP-PRD~ @(JJ @dead))) (. @.))

# tests phrase rules 1 and 14
# tests clause rule 1

# in terminal rule: top of stack is Elvis
# in subword rule: top of stack is type(Elvis, _)
# in word rule 1: top of stack is type(Elvis, _)
# in terminal rule: top of stack is Presley
# in subword rule: top of stack is type(Presley, _)
# in word rule 1: top of stack is type(Presley, _)
# in phrase rule 14: top of stack is type(type(Presley, _)@type(Elvis, _), _)
# in terminal rule: top of stack is is
# in subword rule: top of stack is type(is, _)
# in word rule 1: top of stack is type(is, _)
# in terminal rule: top of stack is dead
# in subword rule: top of stack is type(dead, _)
# in word rule 1: top of stack is type(dead, _)
# in phrase rule 13: top of stack is type(dead, _)
# in phrase rule 1: top of stack is type(type(is, _)@type(dead, _), _)
# in headphrase rule: top of stack is type(type(is, _)@type(dead, _), _)
# in clause rule 1: top of stack is type(type(type(is, _)@type(dead, _), _)@type(type(Presley, _)@type(Elvis, _), _), s)


