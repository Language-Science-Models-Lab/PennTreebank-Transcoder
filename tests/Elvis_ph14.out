[[Elvis, Presley], [is, dead]]:type(type(type(is, _)@type(dead, _), _)@type(°‚O@type(Elvis, _), _), s)
